opapi/src/generators/errors.ts

```diff
@@ -33,6 +33,19 @@ ${Object.entries(codes)
 
 type ErrorCode = typeof codes[keyof typeof codes]
 
+declare const window: any
+type CryptoLib = { getRandomValues(array: Uint8Array): number[] }
+
+let cryptoLib: CryptoLib =
+  typeof window !== 'undefined' && typeof window.document !== 'undefined'
+    ? window.crypto
+      ? window.crypto // Note: On browsers we need to use window.crypto instead of the imported crypto module as the latter is externalized and doesn't have getRandomValues().
```

Comment:
This was added because esbuild seems to rewrite `crypto` to use a broken polyfill instead of the real Crypto browser API (available through `crypto` or `window.crypto`), which results in this runtime error:

```
Module "crypto" has been externalized for browser compatibility. Cannot access "crypto.getRandomValues" in client code
```